### ISSUE TITLE
StatusCode.INVALID_ARGUMENT----- details = "Must provide image parameters" 

### ISSUE THREAD

**Issue by agentx3** [Stability-AI/stability-sdk#87]
I have not touched my application's code but recently within the past hours it has stopped working. I get this error message when I send in request using the python modules
<_MultiThreadedRendezvous of RPC that terminated with:
```
    status = StatusCode.INVALID_ARGUMENT
    details = "Must provide image parameters."
    debug_error_string = "UNKNOWN:Error received from peer ipv4:104.18.21.212:443 {created_time:"2022-10-13T23:50:58.054306143+00:00", grpc_status:3, grpc_message:"Must provide image parameters."}"
>
```

This error shows up regardless of whether or not I provide an `init_image` to the client object.

**Comment by MichaelFerrier** [comment]
I ran into the same problem, so I tried upgrading stability-sdk, and it worked fine after that.

pip install --upgrade stability-sdk

**Comment by willydavid1** [comment]
It's down https://github.com/vpzomtrrfrt/stability-client/issues/9

**Comment by johnsabath** [comment]
Apologies for the breaking change in behavior.  Updating to a newer version of the stability-sdk will resolve this issue, and we'll be putting additional verification checks in place to prevent this from happening again in the future.

**Comment by agentx3** [comment]
> I ran into the same problem, so I tried upgrading stability-sdk, and it worked fine after that.
> 
> pip install --upgrade stability-sdk

I run it in a docker container so I just set the versioning on requirements.txt to 0.2.4 instead, but updating it does work thank you

**Comment by chigozienri** [comment]
> > I ran into the same problem, so I tried upgrading stability-sdk, and it worked fine after that.
> > pip install --upgrade stability-sdk
> 
> I run it in a docker container so I just set the versioning on requirements.txt to 0.2.4 instead, but updating it does work thank you

Note that v0.2.6 is now available, and highly recommended - this fixes a bug introduced in 0.2.5.
